crates/config/src/config.rs

```diff
@@ -221,7 +221,7 @@ fn default_enable_debug_rpc() -> bool {
 
 impl Default for DebugConfig {
     fn default() -> Self {
-        const EXPECTED_TX_UPPER_BOUND_CYCLES: u64 = 70000000u64;
+        const EXPECTED_TX_UPPER_BOUND_CYCLES: u64 = 350000000u64;
```

Comment:
Note that this limit update is the core fix.

Reply:
The fix of withdrawal is also important, it double the count of packaged withdrawals in a block.